### ISSUE TITLE
Similiar project 

### ISSUE THREAD

**Issue by ad-m** [originx/android_emulator_docker#20]
I would like point that there is similiar project: https://medium.com/@AndreSand/android-emulator-on-docker-container-f20c49b129ef
What are pros and cons of that project?

**Comment by originx** [comment]
The projects are in essence the same deal across all docker android emulator images you find. Some have better, some worse quality in terms of picking which API you want, updates, base os etc.

The downside of all docker images at this moment (that I know of) is they do not support L1 nested virtualization.
In layman terms it means:
your system (host) runs docker guest (e.g. ubuntu) that runs accelerated Emulator (x86).

Sadly this currently works well with Linux host machines (kvm) if you run it on a mac or windows you will need to do more to have it setup, which means you need to adjust your scripts accordingly based on your virtualization tool of choice.
for more infos you can refer to these bugstickets:

https://github.com/intel/haxm/issues/51
https://issuetracker.google.com/issues/79499949#comment10

Basically in short if you want x86 docker images to work across all systems try looking into vbox and vmware support for running docker images.
